scripts/visualize/visualize-doy.R

```diff
@@ -96,15 +103,13 @@ visualize.doy <- function(viz = getContentInfo(viz.id = "doy-NM")){
   
   g.zoomer <- xml_add_sibling(xml_children(root)[[length(xml_children(root))]], 'g', id='total_hydro','class'='total_hydro')
```

Comment:
the structure of this part of the svg goes something like this:
```XML
<g id="total_hydro" class="total_hydro">
  <g> ...some axes things in here
    <g id="totalHydro" class="total-hydrograph">
      <polyline class="total-hydrograph" points ="..."/>
     </g>   
  </g>
</g>
```
The classes and the ids might be confusing because of similar names (total_hydro vs total-hydrograph) and the duplication (the parent group has the same class of all of the polyline children).